Makefile

```diff
@@ -0,0 +1,48 @@
+GO_VET_CMD      = go tool vet --all -shadow
+GO_DIRS         = $(shell ls -d */ | \grep -v vendor)
+GO_PKGS         = $(shell go list ./... | \grep -v vendor)
+GO_SRCS         = $(shell find . -type f -name '*.go')
+
+# assets removes the existing bindata.go and regenerates it.
+.PHONY          : assets
+assets          : clean resources/bindata.go
+
+# test runs the lint/vet/fmt and the tests.
+.PHONY          : test
+test            : vet lint fmt imports
+test            : resources/bindata.go ${GO_SRCS}
```

Comment:
I suggest do know touch `resources/bindata.go` file since it is generated by another go library and I don't think we should do a test for it.

Reply:
In here, bindata.go is a dependency, i.e. if it changes, then the test will re-run.

Reply:
I see. Could we rename the resources/bindata.go section? Start a `.PHONY` section and rename to some keywords other than `resources/bindata.go`(The section I mentioned is at the end of Makefile)

Reply:
So you want to remove the `go-bindata` call to generate `resources/bindata.go` and manually generate it instead?

Reply:
No. I don't want you to remove go-bindata call. I just want the following part:

resources/bindata.go    : $(shell find resources -type f -name '*.json')
		rm -f $@
 		go-bindata -o resources/bindata.go -pkg resources resources/
 	        goimports -w resources/bindata.go
		gofmt -w -s resources/bindata.go

to have same `.PHONY` section like what you have done for `fmt` `clean` etc.

Reply:
`resources/bindata.go` is a file, not a PHONY target. Why would it be in the phony section? It would remove the whole purpose of the makefile.